examples/lsm303_accel_inclinometer.py

```diff
@@ -0,0 +1,38 @@
+""" Display inclination data five times per second """
+
+import time
+from math import atan2, degrees
+import board
+import busio
+import adafruit_lsm303_accel
+
+
+i2c = busio.I2C(board.SCL, board.SDA)
+sensor = adafruit_lsm303_accel.LSM303_Accel(i2c)
+
+
+def vector_2_degrees(x, y):
+    radians = atan2(y, x)
+    degrees_calc = degrees(radians)
+    if degrees_calc < 0:
+        degrees_calc = 360 + degrees_calc
+    return degrees_calc
+
+
+def get_inclination(_sensor):
+    return get_inclination_respect_x(_sensor), get_inclination_respect_y(_sensor)
+
+
+def get_inclination_respect_x(_sensor):
+    accel_axis_data = _sensor.acceleration
+    return vector_2_degrees(accel_axis_data[0], accel_axis_data[2])
+
+
+def get_inclination_respect_y(_sensor):
+    accel_axis_data = _sensor.acceleration
+    return vector_2_degrees(accel_axis_data[1], accel_axis_data[2])
+
+
+while True:
+    print("inclination: (%s, %s)" % (get_inclination(sensor)))
```

Comment:
Consider switching to newer style `.format` string formatting instead of `%`.

Also maybe add more description of what the two values are? Something like this?
```python
print("XZ angle = {:6.2}deg   YZ angle = {:6.2}deg".format(get_inclination(sensor)))
```

Reply:
I like it, will make this change tonight.

Reply:
I'm getting this error when I use this code as is. 

    TypeError: can't convert 'tuple' object to str implicitly

I've changed it to this which solves this tuple conversion error by making a variable and accessing the indexes individually. If there is some other syntax that allows it to work as a one liner I can use that but I'm not sure of how to do it. I also used float formatting because it showed exponential notation sometimes.

    inclination = get_inclination(sensor)
    print("XZ angle = {:6.2f}deg   YZ angle = {:6.2f}deg".format(inclination[0],inclination[1]))

Reply:
Ah, yah. My bad. I was just coding from the hip. The actual formatter syntax may be wrong too.

Try this:
```python
print("XZ angle = {:6.2f}deg   YZ angle = {:6.2f}deg".format(*get_inclination(sensor)))
```
Added `f`s to the formatter and used the `*` trick to unpack the tuple. What do you think about splitting it into two lines?
```python
angle_xz, angle_yz = get_inclination(sensor)
print("XZ angle = {:6.2f}deg   YZ angle = {:6.2f}deg".format(angle_xz, angle_yz))
```

Reply:
Ah the asterisk does work. I like your two line version best. I've just pushed a change to that one.